### DISCUSSION TITLE
How can I use a CFD loss function with PyTorch3D?

### DISCUSSION THREAD

**Discussion by cTatu** [facebookresearch/pytorch3d#758] [Ideas]
Hello! My idea is to develop a topology optimization AI for CFD wind tunnel simulations. Basically, starting from a sphere I want the model to create the most lift/drag efficient shape through a number of iterations. 

So can I use as a loss function for example `lift `divided by `drag `coefficient? Or should I use a 3D scalar field for the loss function? If that's the case I don't know how could I pull this off. Any guidelines would be very much appreciated! 

Thank you!

**Comment by nikhilaravi** [comment]
@cTatu this would be a great question for the Discussion section! I will move this issue over!
